samples/drivers/mbox/Kconfig.sysbuild

```diff
@@ -10,7 +10,6 @@ string
 	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
 	default "nrf5340bsim_nrf5340_cpunet" if $(BOARD) = "nrf5340bsim_nrf5340_cpuapp"
 	default "adp_xc7k_ae350" if $(BOARD) = "adp_xc7k_ae350"
-	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "mimxrt595_evk_cm33"
```

Comment:
This seems straight up wrong? Why would the board type for mimxrt595 want to do nrf5340dk_nrf5340_cpunet???

Reply:
Originated in #61910. My best guess here: the choice was made to validate the RT595 as a build only target?